tests/Makefile.tests_common

```diff
@@ -8,6 +8,10 @@ ifneq (,$(wildcard $(CURDIR)/tests*/.))
   endif
 
   DEFAULT_MODULE += test_utils_print_stack_usage
+  # add stack metric printing configuration when testing Kconfig
+  ifeq (1,$(TEST_KCONFIG))
+    KCONFIG_ADD_CONFIG += $(RIOTBASE)/tests/test_print_stack_usage.config
```

Comment:
I think we could just have a single `test_defaults.config` file that enables all the test modules by default when compiling tests.

Reply:
will do!